.teamcity/settings/GradleProfilerPublishing.kt

```diff
@@ -22,6 +24,8 @@ object GradleProfilerPublishing : BuildType({
         param("env.ORG_GRADLE_PROJECT_sdkmanToken", "%gradleprofiler.sdkman.token%")
         param("env.GRADLE_CACHE_REMOTE_USERNAME", "%gradle.cache.remote.username%")
         param("env.GRADLE_CACHE_REMOTE_PASSWORD", "%gradle.cache.remote.password%")
+        param("env.PGP_SIGNING_KEY", "%pgpSigningKey%")
```

Comment:
Personally I like storing these credentials in TC parameters so that they can be updated without changing code, wdyt?

Reply:
That is a good point. I stored it for now like this, since I don't want to define the pgpSigning key in the root project, which would essentially allow all builds to access it. I did the same for native platform. Let's revisit this later.

Reply:
If pgp signing key is used only in gradle-profiler, why don't define it in gradle-profiler TC project?